File: ex09.c
#include<stdio.h>

int prime (int num){
	int i, counter=0;
	for (i=num; i>=1; i--){
		if (num%i==0){
			counter=counter+1;
		}
		
	}
	if (counter>2){
			printf("the number is not prime");
		}
		else {
			printf("the number is prime");
		}
}

int main(void){
	prime(994);
}

File: ex09.py
def check_if_prime (number):
    count=0
    j=number
    while j>=1:
        if (number % j==0):
            count=count+1
        j=j-1
    if count<=2:
        print("the number is prime")
    else:
        print("the number is not prime")
print(check_if_prime(997))